crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -524,28 +505,36 @@ impl Faucet for SimpleFaucet {
     ) -> Result<FaucetReceipt, FaucetError> {
         info!(?recipient, uuid = ?id, "Getting faucet requests");
 
-        let (digest, coin_ids, sent_amounts) = self.transfer_gases(amounts, recipient, id).await?;
-        if coin_ids.len() != amounts.len() {
-            error!(
-                uuid = ?id, ?recipient,
-                "Requested {} coins but got {}",
-                amounts.len(),
-                coin_ids.len()
-            );
-        }
+        let (digest, coin_ids) = self.transfer_gases(amounts, recipient, id).await?;
 
         info!(uuid = ?id, ?recipient, ?digest, "PaySui txn succeeded");
-        Ok(FaucetReceipt {
-            sent: coin_ids
-                .iter()
-                .zip(sent_amounts)
-                .map(|(coin_id, sent_amount)| CoinInfo {
-                    transfer_tx_digest: digest,
-                    amount: sent_amount,
-                    id: *coin_id,
-                })
-                .collect(),
-        })
+        let mut sent = Vec::with_capacity(coin_ids.len());
+        for coin_id in coin_ids {
+            let mut retry_delay = Duration::from_millis(500);
```

Comment:
I tried getting this to work locally, and it wasn't, so I added the delay. But the tests still didn't pass. I'm not sure if it is just something wrong with my local build/invocation.